healthcheck/app.py

```diff
@@ -4,9 +4,13 @@
 
 app = Flask(__name__)
 
-@app.route("/")
+@app.route("/healthcheck/printer")
```

Comment:
need a pr in core v4 to account for this, u can add it to the one u make for led sign health check